canonical/store-components
# change the order of summary and ratings in card component

## Done
changed the order of summary and ratings in card component

## QA
Go to https://store-components-30.demos.haus/?path=/story/packagecard--ratings
See if the ratings come first after the summary.

## Fixes
Fixes: https://warthogs.atlassian.net/browse/WD-2580

## Screenshot
<img width="1188" alt="Screen Shot 2023-03-10 at 9 59 29 AM" src="https://user-images.githubusercontent.com/90341644/224286380-9d971f41-216a-4d68-bc2e-fd00fa9702ba.png">
